sdk/nodejs/ec2/getSubnetIds.ts

```diff
@@ -17,15 +17,13 @@ import * as utilities from "../utilities";
  * import * as pulumi from "@pulumi/pulumi";
  * import * as aws from "@pulumi/aws";
  * 
- * const exampleSubnetIds = pulumi.output(aws.ec2.getSubnetIds({
+ * const exampleSubnetIds = aws.ec2.getSubnetIds({
  *     vpcId: var_vpc_id,
- * }));
+ * });
  * const exampleSubnet: Output<aws.ec2.GETSUBNETResult>[] = [];
```

Comment:
this line is also bad.  where did `GETSUBNETResult` come from instead of `GetSubnetResult`?